src/pages/index.js

```diff
@@ -100,30 +256,39 @@ const addCardPopup = new PopupWithForm({
 });
 addCardPopup.setEventListeners();
 
+const editAvatarPopup = new PopupWithForm({
+  popupSelector: "#change__profile_picture",
+  handleFormSubmit: handleAvatarFormSubmit,
+});
+editAvatarPopup.setEventListeners();
+
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
   addCardFormValidator.toggleButtonState();
 });
 
-const popupWithImage = new PopupWithImage({
-  popupSelector: ".preview__modal",
-});
-popupWithImage.setEventListeners();
-
-const profileEditFormValidator = new FormValidator(options, profileEditForm);
-profileEditFormValidator.enableValidation();
-
-const addCardFormValidator = new FormValidator(options, addCardFormElement);
-addCardFormValidator.enableValidation();
-
-const userInfo = new UserInfo({
-  profileTitle,
-  profileDescription,
-});
-
 profileEditButton.addEventListener("click", () => {
   const userData = userInfo.getUserInfo();
   profileTitleInput.value = userData.name;
   profileDescriptionInput.value = userData.job;
   editProfilePopup.open();
 });
+
+profileAvatarEditButton.addEventListener("click", () => {
+  editAvatarPopup.open();
+});
+
+//like function
+const handleLikeButton = (cardId, cardInstance) => {
+  const isLiked = cardInstance._like;
```

Comment:
`_like`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.

Reply:
* You haven't corrected my remark https://github.com/Alexisct2212/se_project_aroundtheus/pull/1#discussion_r1744543060  Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.